request-cert/k8s_certificates.go

```diff
@@ -22,6 +22,9 @@ import (
 	"time"
```

Comment:
How did you test? Ugh no CI in this repo!

Reply:
Or makefile...

Reply:
Yea, no CI or Makefile. I can add Makefile though,. Maybe I can test locally.